liskov/roster.json

```diff
@@ -33,6 +33,11 @@
       {
         "name": "Anthony Long Yin Chan",
         "github": "anthonychan1211"
-      }
+      },
+      {
+        "name": "Abbie Wills",
+        "github": "AbbieWills"
+      },
```

Comment:
Can you please remove the trailing comma 😄 ?

Reply:
I think i've done it now :)